routes/news.routes.js

```diff
@@ -8,7 +8,7 @@ router.get(
   '/main-news',
   async (req, res) => {
     try {
-      const { dataModel } = await lastItem(newsModel, 3)
+      const { dataModel } = await lastItem(newsModel, '', '', 3)
```

Comment:
if we have more than  3 args, we need to pass object to func args

Reply:
fixed